doc/api/buffer.md

```diff
@@ -81,7 +80,7 @@ to one of these new APIs.*
 
 * [`Buffer.from(array)`] returns a new `Buffer` that *contains a copy* of the
   provided octets.
-* [`Buffer.from(arrayBuffer[, byteOffset [, length]])`][`Buffer.from(arrayBuffer)`]
+* [`Buffer.from(arrayBuffer[, byteOffset[, length]])`][`Buffer.from(arrayBuf)`]
```

Comment:
The whitespace change on this line makes this formatted differently from all our other docs. This change appears to be just to keep things to wrapping at 80 characters. I would undo it and not worry about the 80 character wrapping on this line in this PR. It's good to take care of lines that you are already editing for other reasons, and we often encourage doing so for nearby lines too. But to add 80-char wrapping to the entire doc with this otherwise narrow change is probably not the way to go. For one thing, it will make backporting more difficult.

Reply:
Was this a typo space in this exact case? Space before comma seems uncommon. There were some in this doc, but not everywhere.

Reply:
@vsemozhetbyt Yes, you're right. Not sure about shortening `arrayBuffer` to `arrayBuf` but then again, we already use `buf` elsewhere for a `Buffer` instance, so why not? :-D

Reply:
I'm convinced the whitespace was in error.  If you search https://nodejs.org/api/all.html for ` [,`, you will find 9 instances, corresponding to this function and to `socket.send(msg, [offset, length,] port [, address] [, callback])`.  I think the `socket.send` case has whitespace because both `address` and `callback` are optional, and it would be very confusing to read without the whitespace.

The shortened `arrayBuf` occurs in the link marker, not in the actual body text, so it won't affect the HTML output.  IMHO it's an innocuous change.

Reply:
Yup, I'm on board with this now. Sorry for the noise.